client/Wayland/wlf_disp.c

```diff
@@ -377,3 +377,23 @@ BOOL wlf_disp_uninit(wlfDispContext* wlfDisp, DispClientContext* disp)
 	wlfDisp->disp = NULL;
 	return TRUE;
 }
+
+int wlf_list_monitors(wlfContext* wlc)
+{
+	uint32_t i, nmonitors = UwacDisplayGetNbOutputs(wlc->display);
+
+	for (i = 0; i < nmonitors; i++)
+	{
+		const UwacOutput* monitor = UwacDisplayGetOutput(wlc->display, i);
```

Comment:
is this guaranteed to be `!= NULL`?

Reply:
yes the number of monitors can't change in our back, but you're right I should add a check.

Reply:
@hardening you are adding a check or not? do want to merge this ;)